packages/cln_plugin/test/cln_plugin_method_test.dart

```diff
@@ -80,6 +80,27 @@ void main() {
       expect(plugin.options["greeting"]!.deprecated, false);
     });
 
+    test('Property test - subscriptions', () async {
+      Future<Map<String, Object>> notifyMethod(
+          Plugin plugin, Map<String, Object> request) {
+        plugin.log(level: 'debug', message: 'Notification received!');
+        return Future.value({});
+      }
+
+      var plugin = Plugin(
+          dynamic: true,
+          onInit: (plugin) {
+            return Future.value({});
+          });
+
+      plugin.registerSubscriptions(event: 'connect', callback: notifyMethod);
+      // We added a Subscription, we expect the class property options to be non empty.
+      expect(plugin.subscriptions.isNotEmpty, true);
+
+      // We expect the name of the subscription to be set as the method name
+      expect(plugin.subscriptions['connect']!.method, 'connect');
```

Comment:
```suggestion
       expect(plugin.getManifest['subscriptions'], notNull);
       expect(plugin.getManifest['subscriptions']![0], "connect");
```

`expect(plugin.subscriptions['connect']!.method, 'connect');` is like `assert 1 + 1 == 2` :)

Reply:
oh no! I didnt have anything else to check here, hence this dumb mistake :D